python/tvm/relay/_quantization.py

```diff
@@ -0,0 +1,38 @@
+"""FFI exposing the Relay type inference and checking."""
+from __future__ import absolute_import
+import tvm
+import topi
+from tvm._ffi.function import _init_api
+from .op.op import register_compute, register_schedule, register_pattern, OpPattern
+
+
+@register_compute("simulated_quantize")
+def simulated_quantize_compute(attrs, inputs, output_type, target):
+    """Compiler for simulated_quantize."""
+    assert len(inputs) == 3
+    assert attrs.sign
+    assert attrs.rounding == "round"
+
+    data, scale, bit = inputs
+    sign_bit = 1 if attrs.sign else 0
+
+    return [topi.identity(data)]
+    # TODO: Does tvm support scalar? shape = []
+    print(scale.shape)
+    qscale = tvm.compute([], lambda : topi.power(2.0, (bit - sign_bit)))
```

Comment:
can use shift and cast here

Reply:
pay attention to the data type, as topi may not do automatic type conversion, note that bits is int32